apps/site/assets/css/_header.scss

```diff
@@ -560,24 +560,44 @@ nav.m-menu--mobile {
   }
 }
 
+@mixin focus-visible {
+  &.focus-visible {
+    empty-cells: inherit;
```

Comment:
This is used to "break" the CSS optimizer and force it to keep these rules separate so WebKit doesn't end up choking on comma'd rules with `:focus-visible` in them. I don't _think_ this should have any noticeable affect other than that, but let me know if there's a better property to use here.

Reply:
Hm, this pick of property seems good. Can you please add a comment in the CSS about this too?